fss_check/scripts/daily_fss.py

```diff
@@ -0,0 +1,362 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import argparse
+from datetime import timezone
+from pathlib import Path
+
+import astropy.units as u
+import jinja2
+import matplotlib
+import numpy as np
+from acdc.common import send_mail
```

Comment:
And this seems like a method that has become generally useful and should get moved (from acdc to ska_helpers) but out of scope for this PR.